runtime/rasdump/dmpmap.c

```diff
@@ -121,7 +128,21 @@ static const DgDumpEnvSwitch dgSwitches[] = {
 			}
 };
 
+typedef struct OomDefaultSetting
+{
+	char *typeString;
+	char *onOpt;
+} OomDefaultSetting;
+
+static const OomDefaultSetting oomDefaultTable[] = {
+		{HEAP_KIND, SYSTHROW_OOM_1_4},
+		{JAVA_KIND,  SYSTHROW_OOM_1_4},
+		{SNAP_KIND,  SYSTHROW_OOM_1_4},
+		{SYSTEM_KIND, SYSTHROW_OOM_1_1}
+};
+
 static const int numDumpEnvSwitches = ( sizeof(dgSwitches) / sizeof(DgDumpEnvSwitch) );
+static const int numOomSettings = ( sizeof(oomDefaultTable) / sizeof(OomDefaultSetting) );
```

Comment:
The outer parentheses are not helpful, and the expression `sizeof(oomDefaultTable) / sizeof(OomDefaultSetting)` is better written as `sizeof(oomDefaultTable) / sizeof(oomDefaultTable[0])` so a change to the type of `oomDefaultTable` doesn't break this.

Reply:
Done.

Do you want me to fix the previous line (which follows the same format)?
`static const int numDumpEnvSwitches = ( sizeof(dgSwitches) / sizeof(DgDumpEnvSwitch) );`

Reply:
Seems reasonable to do so while you're here.

Reply:
Done.